### DISCUSSION TITLE
Open browser using parameters?

### DISCUSSION THREAD

**Discussion by Patomanriquezb** [U-C-S/Hurl#177] [General]
Tried to do it but I failed haha. Can it be done with Hurl? How?

For reference, I'm trying to use this chrome feature: `chrome.exe --profile-directory="Profile 1"`

**Comment by U-C-S** [comment]
> Tried to do it but I failed haha

Can you describe how you tried doing it ? I mean, paste the UserSettings.json with what you have tried. I can help you with that

If you are asking how to do it, You can checkout, an example UserSettings.json in the docs - https://github.com/U-C-S/Hurl/tree/main/Docs#user-settings which also has config for chrome profiles (Notice the escape character `\` in the example)

**Reply by Patomanriquezb** [reply]
Don't know what I'm doing wrong :( I don't get the incognito option, I right click chrome and nothing pops up.

```
"Browsers": [
    {
      "Name": "Google Chrome",
      "ExePath": "C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe",
      "AlternateLaunches": [
        {
          "ItemName": "Incognito",
          "LaunchArgs": "-incognito"
        }
      ]
    },
    {
      "Name": "Microsoft Edge",
      "ExePath": "C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe"
    },
    {
      "Name": "Firefox",
      "ExePath": "C:\\Program Files\\Mozilla Firefox\\firefox.exe"
    }
  ],
```

**Reply by Patomanriquezb** [reply]
NVM, just had to restart.
Thanks!

**Reply by U-C-S** [reply]
> just had to restart.

Looks like I did not add that to the docs. Sorry for the trouble here
